packages/xrpl/snippets/src/claimPayChannel.ts

```diff
@@ -4,7 +4,8 @@ import {
   PaymentChannelCreate,
   PaymentChannelClaim,
   hashes,
-} from '../../dist/npm'
+} from '../../src'
+// Replace this import with "from 'xrpl'" if you copy and paste this code
```

Comment:
Should this be tested differently in CI and actually use `xrpl`?

Reply:
That could be good since we want these to be copy + pastable. The only time that should cause problems is if we make breaking changes since the interface for the snippets would be incorrect, but that's decently rare. I'll open a ticket for it.